package.json

```diff
@@ -164,7 +164,7 @@
 		"whatwg-fetch": "1.1.1"
 	},
 	"devDependencies": {
-		"@automattic/calypso-build": "1.0.0-alpha.0",
+		"@automattic/calypso-build": "1.0.0-alpha.2",
 		"@automattic/calypso-color-schemes": "1.0.0-alpha.1",
```

Comment:
Is this `@automattic/calypso-color-schemes` anymore used in Jetpack directly as it's now also dependency in calypso-build? https://github.com/Automattic/jetpack/pull/11941/files#diff-8ee2343978836a779dc9f8d6b794c3b2R10

Reply:
Hmm, too many tabs open. Meant to comment at https://github.com/Automattic/jetpack/pull/11802 :-)

Reply:
I'll remove it there.